assets/vue/components/import-modal.vue

```diff
@@ -9,35 +9,43 @@
-							<ul class="features">
+							<template v-if="! ( Object.keys( this.siteData ).includes( 'template_name' ) )">
+								<h4>{{strings.general}}:</h4>
+								<ul v-if="! ( Object.keys( this.siteData ).includes( 'template_name' ) )" class="features">
+									<li class="option_toggle">
+										<label class="option-toggle-label"
+												:class="importOptions.content ? 'active' : 'inactive'"><span
+												class="dashicons dashicons-admin-post"></span><span>{{strings.content}}</span></label>
+										<toggle-button @change="adjustImport( 'content' )" :value="importOptions.content"
+												color="#28da00"></toggle-button>
+									</li>
 								<li class="option_toggle">
-									<label class="option-toggle-label"
-											:class="importOptions.content ? 'active' : 'inactive'"><span
-											class="dashicons dashicons-admin-post"></span><span>{{strings.content}}</span></label>
-									<toggle-button @change="adjustImport( 'content' )" :value="importOptions.content"
-											color="#28da00"></toggle-button>
-								</li>
-							<li class="option_toggle">
-									<label class="option-toggle-label"
-											:class="importOptions.customizer ? 'active' : 'inactive'"><span
-											class="dashicons dashicons-admin-customizer"></span><span>{{strings.customizer}}</span></label>
-									<toggle-button @change="adjustImport( 'customizer' )" :value="importOptions.customizer"
-											color="#28da00"></toggle-button>
-								</li>
-							<li class="option_toggle">
-									<label class="option-toggle-label"
-											:class="importOptions.widgets ? 'active' : 'inactive'"><span
-											class="dashicons dashicons-admin-generic"></span><span>{{strings.widgets}}</span></label>
-									<toggle-button @change="adjustImport( 'widgets' )" :value="importOptions.widgets"
-											color="#28da00"></toggle-button>
-								</li>
-							</ul>
-							<hr>
+										<label class="option-toggle-label"
+												:class="importOptions.customizer ? 'active' : 'inactive'"><span
+												class="dashicons dashicons-admin-customizer"></span><span>{{strings.customizer}}</span></label>
+										<toggle-button @change="adjustImport( 'customizer' )" :value="importOptions.customizer"
+												color="#28da00"></toggle-button>
+									</li>
+								<li class="option_toggle">
+										<label class="option-toggle-label"
+												:class="importOptions.widgets ? 'active' : 'inactive'"><span
+												class="dashicons dashicons-admin-generic"></span><span>{{strings.widgets}}</span></label>
+										<toggle-button @change="adjustImport( 'widgets' )" :value="importOptions.widgets"
+												color="#28da00"></toggle-button>
+									</li>
+								</ul>
+								<hr>
+							</template>
 							<h4>{{strings.plugins}}:</h4>
 							<ul class="features">
+								<li v-if="siteData.mandatory_plugins" class="option_toggle" v-for="( plugin, index ) in siteData.mandatory_plugins">
```

Comment:
This can be simplified. Be sure to return an empty array on the backend for the mandatory plugins if none should be added. Then just do a v-for which implicitly loops over a 0 length array and nothing is shown.